drivers/shared/src/make_dmy.c

```diff
@@ -29,10 +29,6 @@
 #include <vic_run.h>
 #include <vic_driver_shared.h>
 
-#ifndef _LEAPYR
-#define LEAPYR(y) (!((y) % 400) || (!((y) % 4) && ((y) % 100)))
-#endif
-
 /******************************************************************************
```

Comment:
Do we no longer check for leap year or has this been moved somewhere else (I am getting a bit lost in the single commit - it's big!)